docs/en/api/POSIX-API.md

```diff
@@ -169,6 +171,26 @@ pid mount_point alluxio_path
 
 ## Advanced Setup
 
+### Select which libfuse version to use
+
+Alluxio now supports both libfuse2 and libfuse3. 
+
+If only one version of libfuse is installed, that version is used. In most distros, libfuse2 and libfuse3 can coexist. If both versions are installed, **libfuse2** will be used by default (for backward compatibility). 
+
+Or, you can manually determine which version to use by changing configuration in `${ALLUXIO_HOME}/conf/alluxio-site.properties`. 
```

Comment:
To set a version explicitly, add the following configuration....

Reply:
the example may be better to set use `libfuse.version=3` since it's not the default behavior

Reply:
updated